src/platform.ts

```diff
@@ -20,6 +20,28 @@ export enum Platform {
     Ubuntu16
 }
 
+function getValue(name: string, lines: string[]) {
```

Comment:
Now that this function is not in getCurrentPlatform, you might want to give it a better name as getValue is not very descriptive.